packages/lockfile-lint/src/config.js

```diff
@@ -45,6 +45,13 @@ module.exports = (argv, exitProcess = false, searchFrom = process.cwd()) => {
         type: 'boolean',
         describe: 'validates the use of HTTPS as protocol schema for all resources'
       },
+      n: {
+        alias: ['validate-package-names'],
+        type: 'boolean',
+        describe:
+          "validates that the resource URL specifies the same package name as that listed as the lockfile entry's key",
+        implies: 'allowed-hosts'
```

Comment:
@lirantal Yargs now enforces that if package names are validated, the developer has also specified specifically allowed hosts. All hosts I know of follow the checked convention (i.e., Yarn, NPM, Verdaccio). Some might not, but that is out-of-scope for this PR. At least now, users who _don't_ specify a host and use non-standard sources (e.g., Github, etc.) will get a friendly message notifying them that package name validation only works when particular hosts have been chosen.